PluginsAndFeatures/azure-toolkit-for-intellij/src/com/microsoft/azure/hdinsight/serverexplore/ui/AddNewClusterForm.form

```diff
@@ -161,16 +161,16 @@
                         </constraints>
                         <properties>
                           <labelFor value="7bd75"/>
-                          <text value="  Server:"/>
+                          <text value="  Server(Required):"/>
                         </properties>
                       </component>
                       <component id="34c8" class="javax.swing.JLabel">
                         <constraints>
-                          <grid row="1" column="0" row-span="1" col-span="1" vsize-policy="0" hsize-policy="0" anchor="4" fill="0" indent="5" use-parent-layout="false"/>
+                          <grid row="1" column="0" row-span="1" col-span="1" vsize-policy="0" hsize-policy="0" anchor="4" fill="0" indent="0" use-parent-layout="false"/>
```

Comment:
Is this indent changes(and following) required?

Reply:
@wezhang Wei, yes, adding [Required] text on label will make the cell width expanded, change indent can reduce the width.